src/Element/BaseInput.php

```
@@ -62,6 +62,11 @@ abstract class BaseInput extends BaseTag
     use HasTranslate;
     use HasType;
 
+    /**
+     * Suffix appended to the element `id` when `aria-describedby` is set to `true`.
+     */
+    protected string $ariaDescribedBySuffix = '';
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $ariaDescribedBySuffix. Keep variable name length under 20. (undefined)

(LongVariable)